test/unit/AccountClient.test.ts

```diff
@@ -76,7 +76,7 @@ describe("AccountClient", () => {
 
             it("promise error", () => {
                 client = new postmark.AccountClient("testToken");
-                sandbox.stub(BaseClient.prototype, <any> "httpRequest").yields(undefined, {statusCode: 401, body: 'response'});
+                sandbox.stub(BaseClient.prototype, <any> "httpRequest").rejects({response: {status: 401, data: 'response'}});
```

Comment:
I kind of want to avoid library explicitly relying on request library added (axios), since it introduces not needed dependency, but will think about this one.

Reply:
got rid of privates. Actually sinon made it easy to stub axios, grazias